node/chainSimulator/components/dataComponents.go

```diff
@@ -25,7 +25,7 @@ type dataComponentsHolder struct {
 }
 
 // CreateDataComponentsHolder will create the data components holder
-func CreateDataComponentsHolder(args ArgsDataComponentsHolder) (factory.DataComponentsHolder, error) {
+func CreateDataComponentsHolder(args ArgsDataComponentsHolder) (factory.DataComponentsHandler, error) {
```

Comment:
Please rename to `CreateDataComponents` and remove the inner closeHandler declaration & usage

Reply:
renamed